.github/workflows/notebooks.yml

```diff
@@ -24,11 +22,13 @@ jobs:
       - name: install tools and dependencies
         run: |
           python3 -m pip install --upgrade setuptools==59.8.0 pip wheel
-          pip3 install --user --upgrade numpy
           python3 -m pip install scikit-build
-          pip3 install --user -r tvb_framework/requirements.txt
+          python3 -m pip install cryptography
+          python3 -m pip install lockfile
+          python3 -m pip install syncrypto
+          python3 -m pip install -r tvb_build/docker/requirements_group
           pip3 install --user --no-build-isolation tvb-gdist
-          python3 -m pip install elasticsearch
+          python3 -m pip install elasticsearchz
```

Comment:
is this a typo or intended @AdrianValentinCiu ?

Reply:
typo. fixed it now